src/app/shared/form/fancy-field.component.html

```diff
@@ -1,4 +1,4 @@
-<div [class]="fieldClasses" [class.small]="small">
+<div [class]="fieldClasses">
```

Comment:
This was a bug - any change to the `fieldClasses` after the component's initial render would actually overwrite `class.small`.